src/interface/icons/Armory.tsx

```diff
@@ -0,0 +1,15 @@
+import { t } from '@lingui/macro';
+import React from 'react';
+
+import ArmoryLogo from './images/Armory-logo.png';
+
+const Icon = (props: any) => (
```

Comment:
when typing a wrapper component like this, you should use something like:

```suggestion
type Props = Omit<React.ComponentPropsWithoutRef<'img'>, "src" | "alt" | "className">;
const Icon = (props: Props) => (
```

that way users of the icon get direct typechecking support

Reply:
Thanks for the feedback - I just pushed a change with proper typing.

It looks like DEFAULT_BUILD.tsx uses ArmoryIcon without any text afterwards.  I'm not quite sure what you mean by shortcut - I'm pretty new to React so maybe that's something I haven't come across yet.

Thanks!

Reply:
By a shortcut i just meant a component that had the text and `<Trans>...</Trans>` element as part of the output, but since it is sometimes used without that part it doesn't need it. Thanks!